### ISSUE TITLE
securemof doc should say that we required the mof be generated using '"Microsoft RSA SChannel Cryptographic Provider"' provider

### ISSUE THREAD

**Issue by TravisEz13** [PowerShell/PowerShell-Docs#248]
Example self-sign certificate generation that works:

```PowerShell
    $cert = New-SelfSignedCertificate -Type DocumentEncryptionCert -DnsName 'DscEncryptionCert' -Provider "Microsoft RSA SChannel Cryptographic Provider"
```
does not work:

```PowerShell
    $cert = New-SelfSignedCertificate -Type DocumentEncryptionCert -DnsName 'DscEncryptionCert' 
```

**Comment by PlagueHO** [comment]
@TravisEz13 - Shall I make this change?

**Comment by TravisEz13** [comment]
@PlagueHO Eric is working on it internally.  We are discussing how to properly message this.

**Comment by eslesar** [comment]
Addressed with #265.
